doc/rst/developer/bestPractices/TestSystem.rst

```diff
@@ -606,13 +608,41 @@ suite(s) in which your graph appears, and you should see data for it.
 (Note that for a new graph with only one day of data, it can be hard
 to see the singleton points at first).
 
+Multilocale Performance Testing
++++++++++++++++++++++++++++++++
+Writing a performance test for multilocale setting is similar to single locale
+counterpart. However, helper file suffixes must be as follows.
+
+
+===================== ==================== 
+ Single Locale         Multilocale         
+===================== ==================== 
+ ``.numlocales``       ``.ml-numlocales``  
+ ``.perfexecopts``     ``.ml-execopts``    
+ ``.perfcompopts``     ``.ml-compopts``    
+ ``.perfkeys``         ``.ml-keys``        
+ ``.graph``            ``.ml-perf.graph``  
+ ``.execenv``          ``.ml-execenv``     
+===================== ==================== 
+
+Graph files for multilocale performance tests are listed in ``ML-GRAPHFILES``
+instead of ``GRAPHFILES``.
+
+Finally to run a multilocale performance test ``start_test --perflabel ml-``
+must be used.
+
+Multilocale Communication Counts Testing
+++++++++++++++++++++++++++++++++++++++++
+Writing a communication counts test for multilocale setting is similar to the
+performance counterpart. However, for helper files ``cc-`` label is used instead
+of ``ml-``.
```

Comment:
We don't already have a section describing communication counts testing, and I'm not familiar with comm counts testing at all in single locale.  Should this just be describing communication counts testing in general (rather than specifically for multilocale)?  Is there a way to distinguish between single locale communication counts and multilocale communication counts?

Reply:
You can't have communication when you only have a single locale

Reply:
That makes sense - I think to someone reading the document without this context, the current wording would imply that it was possible.  Maybe instead of saying "for multilocale setting", I would add a note saying that it is only applicable in multilocale settings?